### ISSUE TITLE
Implement removal of files after a certain period

### ISSUE THREAD

**Issue by wasabideveloper** [geometalab/osmaxx-frontend#139]
Implementation of #27.

**Comment by hixi** [comment]
depends on #279.

**Comment by wasabideveloper** [comment]
Really? I don't think so. I think this is a cron job removing the download files after a certain period. I think #279 is about the removal of the conversion result files on converter service side. Or am I wrong?

**Comment by hixi** [comment]
No, this is correct, but not obvious: the files on the conversion manager need to be able to be deleted, otherwise a cronjob makes no sense (the osmaxx-client should issue a delete in case of successful downloaded file, we don't want to store them at two places). I agree, that this alone might warrant a new issue, but then the dependency of this issue should probably be on that one.

On the file removing topic: 
cronjob in a docker-container is not going to be something we even want to think about (couple of tries didn't work - I think this is on purpose, but couldn't find any [reference](http://stackoverflow.com/questions/26822067/running-cron-python-jobs-within-docker) as why this is the case). But having some kind of 'scheduler' would mean another container (with a heartbeat or something), which I'd like to avoid. My suggestion is loosely based on a discussion I had with @das-g a while ago:

Every-time someone downloads a file, the date (and possibly time) is recorded. If a new file conversion is being requested, and the corresponding file of the newly requested conversion is being downloaded (which happens upon notification through an `internal` call from the conversion-service to the osmaxx-webapp), it removes the oldest files, so that no more than, say 10, already downloaded files are being kept in storage. An alternative would be to do the cleanup as stated above, but in fact date-based (ie. so removing after X days is possible).
Either way we wouldn't need a scheduler, which to me seems like a (major) win.

**Comment by wasabideveloper** [comment]
:+1:
For now I suggest to implement the easier variant (remove files after a certain period) and move the check if downloaded or not to the next project.
